molecule/default/tasks/template.yml

```diff
@@ -0,0 +1,154 @@
+---
+- block:
+    - set_fact:
+        template_namespace: template-test
+
+    - name: Ensure namespace exists
+      k8s:
+        definition:
+          apiVersion: v1
+          kind: Namespace
+          metadata:
+            name: "{{ template_namespace }}"
+
+    - name: Check if k8s_service does not inherit parameter
+      community.kubernetes.k8s_service:
+        template: "../templates/pod_template_one.j2"
```

Comment:
Will this automatically look in the `templates` directory as the `template` module would? If so, could we test that?

Reply:
Yes, templates under `templates` directory will be detected.